Riot/Modules/Home/AllChats/ReviewSessionAlertSnoozeController.swift

```diff
@@ -0,0 +1,49 @@
+// 
+// Copyright 2022 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+@objcMembers
+class ReviewSessionAlertSnoozeController: NSObject {
+    
+    private let userDefaults: UserDefaults
```

Comment:
I think the 2 following `init` methods will be useless if you instantiate `userDefaults` value right here.